packages/database/e2e/query/get.modular.e2e.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2016-present Invertase Limited & Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this library except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+const { PATH, CONTENT, seed, wipe } = require('../helpers');
+
+const TEST_PATH = `${PATH}/once`;
+
+describe('get()', function () {
+  before(function () {
+    return seed(TEST_PATH);
```

Comment:
Does it lack the v8 part then the modular part?

Reply:
This is equivalent to `once('value')`. V9 does not have a once function, instead a get function. So I moved it to it's own file